utils.py

```diff
@@ -0,0 +1,164 @@
+"""A collection of utility functions (mostly file reading/writing) used
+in the PathCORE analysis scripts"""
+
+import os
+
+import pandas as pd
+
+
+def load_expression_dataset(path_to_file):
+    """
+    Parameters
+    -----------
+    path_to_file : str
+        The path to the expression dataset.
+        Currently expects a tab-delimited file with row and column names,
+        where the rows are the genes and the columns are the samples.
+
+    Returns
+    -----------
+    pandas.DataFrame
+    """
+    expression_dataset = pd.read_table(path_to_file, header=0)
```

Comment:
`pd.read_table(path_to_file, header=0, index_col=0)` will remove the need for line 23 and 24